#다음중 오류가 발생하는 경우를 고르고 이유를 설명해보자.
'''
a = dict()
1.a['name'] = 'python'
2.a[('a',)] = 'python'
3.a[[1]] = 'python'
4.a[250] = 'python'''''

#3.키는 변할수 없는 값이기에 리스트가 올 수 없다.